Только в /root/dk_scripts/tmp/1: 2-Oct-2024015343LINK-I-Upgi27Ksw-sever-2-2_10.9.1.6.conf
Только в /root/dk_scripts/tmp/1: 5-Jan-2016152945LINK-W-Downfa41KTCS1-SW3-48_192.168.22.75.conf
Только в /root/dk_scripts/tmp/2: sw-sever-2-2_10.9.1.6.conf
diff -r /root/dk_scripts/tmp/2/TBR21-SW3-52_192.168.22.80.conf /root/dk_scripts/tmp/1/TBR21-SW3-52_192.168.22.80.conf
344d343
<  switchport trunk native vlan 66 
Только в /root/dk_scripts/tmp/2: TCS1-SW3-48_192.168.22.75.conf
